Short-Answer/Algorithms_Answers.md

```diff
@@ -2,11 +2,12 @@ Add your answers to the Algorithms exercises here.
 ## Exercise I
 
 ```
-a)  a = 0
+a)  a = 0 // 0(1)
     while (a < n * n * n) // 0(1) * 0(n) * 0(n) = 0(n)
       a = a + n * n // 0(1)
+                  
 ```
-- The loop is 0(n) as it doesn't modifies n, every loop just increases and 0(n) terms dominates 0(1).
+- The loop is "0(n)" as it doesn't modifies n, every loop just increases and 0(n) terms dominates 0(1).
```

Comment:
It actually is linear time, but that has to do with the fact that the inner function within the loop cancels out n^2 of the n^3, making it overall just simply linear time.